communication-refactored/src/actor.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg(feature = "actor")]
+
+pub mod messages;
```

Comment:
```suggestion
#![cfg(feature = "actor")]
pub mod messages;

```

The `cfg` only applies to the single line, not the whole module, right?

Reply:
It was meant for the whole module. 
But I can actually remove that completely, since I already have in my lib.rs:
```
#[cfg(feature = "actor")]
pub mod actor;
```